src/display/api.js

```diff
@@ -807,9 +807,10 @@ class PDFDocumentProxy {
  *
  * @typedef {Object} GetViewportParameters
  * @property {number} scale - The desired scale of the viewport.
- * @property {number} rotation - (optional) The desired rotation, in degrees, of
- *   the viewport. If omitted it defaults to the page rotation.
- * @property {boolean} dontFlip - (optional) If true, the y-axis will not be
+ * @property {number} [rotation] - (optional) The desired rotation,
+ *   in degrees, of the viewport. If omitted it defaults to the page
+ *   rotation.
```

Comment:
Doesn't this fit on the previous line now?

Reply:
Fixed.